include/sys/zfs_ioctl.h

```diff
@@ -338,6 +339,11 @@ typedef struct zinject_record {
 #define	ZEVENT_SEEK_START	0
 #define	ZEVENT_SEEK_END		UINT64_MAX
 
+/* Note percentage use case has a higher resolution */
+#define	zi_percentage	zi_freq
+#define	ZI_PERCENTAGE_MIN	4294UL
```

Comment:
Ah, interesting.  I had originally started using the -f and its 100 value range but realized right away that it wasn't enough for our needs.  I like the idea of just extending -f to have more precision and [1..100] is still backwards compatible!